src/clr-angular/forms/combobox/option-items.directive.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { NgForOf, NgForOfContext } from '@angular/common';
+import {
+  Directive,
+  DoCheck,
+  Input,
+  IterableDiffer,
+  IterableDiffers,
+  TemplateRef,
+  TrackByFunction,
+  ViewContainerRef,
+  OnDestroy,
+} from '@angular/core';
+
+import { OptionSelectionService } from './providers/option-selection.service';
+import { Subscription } from 'rxjs';
+import { ClrPopoverPositionService } from '../../utils/popover/providers/popover-position.service';
+
+@Directive({
+  selector: '[clrOptionItems][clrOptionItemsOf]',
+})
+export class ClrOptionItems<T> implements DoCheck, OnDestroy {
+  private iterableProxy: NgForOf<T>;
+  private _rawItems: T[];
+  private filteredItems: T[];
+  private subscriptions: Subscription[] = [];
+  private filter = '';
+  private _filterField: string;
+  private differ: IterableDiffer<T> | null = null;
+
+  @Input('clrOptionItemsOf')
+  public set rawItems(items: T[]) {
+    this._rawItems = items ? items : [];
+    this.updateItems();
+  }
+
+  @Input('clrOptionItemsTrackBy')
+  set trackBy(value: TrackByFunction<T>) {
+    this.iterableProxy.ngForTrackBy = value;
+  }
+
+  @Input('clrOptionItemsField')
+  public set field(field: string) {
```

Comment:
I'm not seeing a test for this setter that makes sure the correct value is set in this class and in the optionService.

Reply:
The service gets checked in the 'sets display field' test.
I'll add the local field check to this test too.
Fixed.